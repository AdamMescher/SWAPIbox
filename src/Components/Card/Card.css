.card {
  background-color: #747a80;
  height: 250px;
  width: 180px;
  margin: 0 10px 10px 0;
  display: grid;
  grid-template-rows: 20% ;
  grid-template-columns: 80% 20%;
}

.card-header {
  grid-area: 1 / 1 / 2 / 3;
  background-color: magenta;
  display: grid;
  grid-template-rows: 1fr;
  grid-template-columns: 70% 30%;

  border-bottom: 1px white dotted;
}

.card-title {
  color: white;
  height: 100%;
  grid-area: 1 / 1 / 2 / 2;
  justify-content: center;
  text-align: center;
}

.card-favorite-button {
  grid-area: 1 / 2 / 2 / 3;
  height: 100%;
  width: 100%;
  background-color: #4a5360;
  border: none;
  background-color: teal;
  padding: 0;
}

.card-favorite-button:hover {
  color: #ffff00;
}

.card-definition-list {
  grid-area: 2 / 1 / 3 / 3;
  margin: 10px 10px 0 10px;
}
