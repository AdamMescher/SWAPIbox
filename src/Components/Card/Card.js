import React from 'react';
import PropTypes from 'prop-types';
import Button from '../Button/Button';

const Card = ( {name, url, displayData } ) => {
  const mappedNounObject = Object.keys( displayData ).map( key => {
    return ([
        <dt className="card-definition-term">{key}</dt>,
        <dd>{displayData[key]}</dd>
    ])
  });

  return (
    <div className="card">
      <section className="card-header">
        <h3 className="card-title">{name}</h3>
        <Button
        className={url} />
      </section>
      <dl className="card-definition-list">
        {mappedNounObject}
      </dl>
    </div>
  )
}

export default Card;
