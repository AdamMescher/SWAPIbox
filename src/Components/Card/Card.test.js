import React from 'react';
import { shallow, mount } from 'enzyme';
import Card from './Card';

describe( `CARD`, () => {
  let
    wrapperPerson,
    wrapperPlanet,
    wrapperVehicle,
    card;

  beforeEach( () => {
    wrapperPerson = shallow(
      <Card
        nounObject={ {
          name: 'Luke Skywalker',
          species: 'Human',
          homeworld: 'Tatooine',
          language: 'Galactic Basic',
          population: 200000
        } }
      />);

    wrapperPlanet = shallow(
      <Card
        nounObject={{
          name: 'Tatooine',
          terrain: 'desert',
          population: 200000,
          climate: 'arid',
          residents: [
            "Luke Skywalker",
            "C-3PO",
            "Darth Vader",
            "Owen Lars",
            "Beru Whitesun lars",
            "R5-D4",
            "Biggs Darklighter",
            "Anakin Skywalker",
            "Shmi Skywalker",
            "Cliegg Lars"
          ]
        }}
      />);
  });

  wrapperVehicle = shallow(
    <Card
      nounObject={ {
        name: 'Sand Crawler',
        model: 'Digger Crawler',
        class: 'wheeled',
        passengers: 30
      } }
    />);

  it( `should exist`, () => {

  });
  it( ``, () => {});
  it( ``, () => {});
  it( ``, () => {});
});
