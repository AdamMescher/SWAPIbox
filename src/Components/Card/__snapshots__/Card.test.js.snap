// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CARD Matches person snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): ShallowWrapper {
    "length": 1,
    Symbol(enzyme.__root__): [Circular],
    Symbol(enzyme.__unrendered__): <Card
      nounObject={
            Object {
                  "homeworld": "Tatooine",
                  "language": "Galactic Basic",
                  "name": "Luke Skywalker",
                  "population": 200000,
                  "species": "Human",
                }
      }
/>,
    Symbol(enzyme.__renderer__): Object {
      "batchedUpdates": [Function],
      "getNode": [Function],
      "render": [Function],
      "simulateEvent": [Function],
      "unmount": [Function],
    },
    Symbol(enzyme.__node__): Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="card-header"
>
            <h3
                        className="card-title"
            />
            <Button />
</section>,
          <dl
            className="card-definition-list"
>
            <dt
                        className="card-definition-term"
            >
                        name
            </dt>
            <dd>
                        Luke Skywalker
            </dd>
            <dt
                        className="card-definition-term"
            >
                        species
            </dt>
            <dd>
                        Human
            </dd>
            <dt
                        className="card-definition-term"
            >
                        homeworld
            </dt>
            <dd>
                        Tatooine
            </dd>
            <dt
                        className="card-definition-term"
            >
                        language
            </dt>
            <dd>
                        Galactic Basic
            </dd>
            <dt
                        className="card-definition-term"
            >
                        population
            </dt>
            <dd>
                        200000
            </dd>
</dl>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="card-title"
/>,
              <Button />,
            ],
            "className": "card-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": "card-title",
              },
              "ref": null,
              "rendered": null,
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              Array [
                <dt
                  className="card-definition-term"
>
                  name
</dt>,
                <dd>
                  Luke Skywalker
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  species
</dt>,
                <dd>
                  Human
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  homeworld
</dt>,
                <dd>
                  Tatooine
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  language
</dt>,
                <dd>
                  Galactic Basic
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  population
</dt>,
                <dd>
                  200000
</dd>,
              ],
            ],
            "className": "card-definition-list",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "name",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "name",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Luke Skywalker",
              },
              "ref": null,
              "rendered": "Luke Skywalker",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "species",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "species",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Human",
              },
              "ref": null,
              "rendered": "Human",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "homeworld",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "homeworld",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Tatooine",
              },
              "ref": null,
              "rendered": "Tatooine",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "language",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "language",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Galactic Basic",
              },
              "ref": null,
              "rendered": "Galactic Basic",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "population",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "population",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": 200000,
              },
              "ref": null,
              "rendered": 200000,
              "type": "dd",
            },
          ],
          "type": "dl",
        },
      ],
      "type": "div",
    },
    Symbol(enzyme.__nodes__): Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <section
              className="card-header"
>
              <h3
                            className="card-title"
              />
              <Button />
</section>,
            <dl
              className="card-definition-list"
>
              <dt
                            className="card-definition-term"
              >
                            name
              </dt>
              <dd>
                            Luke Skywalker
              </dd>
              <dt
                            className="card-definition-term"
              >
                            species
              </dt>
              <dd>
                            Human
              </dd>
              <dt
                            className="card-definition-term"
              >
                            homeworld
              </dt>
              <dd>
                            Tatooine
              </dd>
              <dt
                            className="card-definition-term"
              >
                            language
              </dt>
              <dd>
                            Galactic Basic
              </dd>
              <dt
                            className="card-definition-term"
              >
                            population
              </dt>
              <dd>
                            200000
              </dd>
</dl>,
          ],
          "className": "card",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3
                  className="card-title"
/>,
                <Button />,
              ],
              "className": "card-header",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                  "className": "card-title",
                },
                "ref": null,
                "rendered": null,
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "section",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                Array [
                  <dt
                    className="card-definition-term"
>
                    name
</dt>,
                  <dd>
                    Luke Skywalker
</dd>,
                ],
                Array [
                  <dt
                    className="card-definition-term"
>
                    species
</dt>,
                  <dd>
                    Human
</dd>,
                ],
                Array [
                  <dt
                    className="card-definition-term"
>
                    homeworld
</dt>,
                  <dd>
                    Tatooine
</dd>,
                ],
                Array [
                  <dt
                    className="card-definition-term"
>
                    language
</dt>,
                  <dd>
                    Galactic Basic
</dd>,
                ],
                Array [
                  <dt
                    className="card-definition-term"
>
                    population
</dt>,
                  <dd>
                    200000
</dd>,
                ],
              ],
              "className": "card-definition-list",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "name",
                  "className": "card-definition-term",
                },
                "ref": null,
                "rendered": "name",
                "type": "dt",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Luke Skywalker",
                },
                "ref": null,
                "rendered": "Luke Skywalker",
                "type": "dd",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "species",
                  "className": "card-definition-term",
                },
                "ref": null,
                "rendered": "species",
                "type": "dt",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Human",
                },
                "ref": null,
                "rendered": "Human",
                "type": "dd",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "homeworld",
                  "className": "card-definition-term",
                },
                "ref": null,
                "rendered": "homeworld",
                "type": "dt",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Tatooine",
                },
                "ref": null,
                "rendered": "Tatooine",
                "type": "dd",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "language",
                  "className": "card-definition-term",
                },
                "ref": null,
                "rendered": "language",
                "type": "dt",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Galactic Basic",
                },
                "ref": null,
                "rendered": "Galactic Basic",
                "type": "dd",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "population",
                  "className": "card-definition-term",
                },
                "ref": null,
                "rendered": "population",
                "type": "dt",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": 200000,
                },
                "ref": null,
                "rendered": 200000,
                "type": "dd",
              },
            ],
            "type": "dl",
          },
        ],
        "type": "div",
      },
    ],
    Symbol(enzyme.__options__): Object {
      "adapter": ReactSixteenAdapter {
        "options": Object {
          "enableComponentDidUpdateOnSetState": true,
        },
      },
    },
  },
  Symbol(enzyme.__unrendered__): null,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section
          className="card-header"
>
          <h3
                    className="card-title"
          />
          <Button />
</section>,
        <dl
          className="card-definition-list"
>
          <dt
                    className="card-definition-term"
          >
                    name
          </dt>
          <dd>
                    Luke Skywalker
          </dd>
          <dt
                    className="card-definition-term"
          >
                    species
          </dt>
          <dd>
                    Human
          </dd>
          <dt
                    className="card-definition-term"
          >
                    homeworld
          </dt>
          <dd>
                    Tatooine
          </dd>
          <dt
                    className="card-definition-term"
          >
                    language
          </dt>
          <dd>
                    Galactic Basic
          </dd>
          <dt
                    className="card-definition-term"
          >
                    population
          </dt>
          <dd>
                    200000
          </dd>
</dl>,
      ],
      "className": "card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3
              className="card-title"
/>,
            <Button />,
          ],
          "className": "card-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": "card-title",
            },
            "ref": null,
            "rendered": null,
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            Array [
              <dt
                className="card-definition-term"
>
                name
</dt>,
              <dd>
                Luke Skywalker
</dd>,
            ],
            Array [
              <dt
                className="card-definition-term"
>
                species
</dt>,
              <dd>
                Human
</dd>,
            ],
            Array [
              <dt
                className="card-definition-term"
>
                homeworld
</dt>,
              <dd>
                Tatooine
</dd>,
            ],
            Array [
              <dt
                className="card-definition-term"
>
                language
</dt>,
              <dd>
                Galactic Basic
</dd>,
            ],
            Array [
              <dt
                className="card-definition-term"
>
                population
</dt>,
              <dd>
                200000
</dd>,
            ],
          ],
          "className": "card-definition-list",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "name",
              "className": "card-definition-term",
            },
            "ref": null,
            "rendered": "name",
            "type": "dt",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "Luke Skywalker",
            },
            "ref": null,
            "rendered": "Luke Skywalker",
            "type": "dd",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "species",
              "className": "card-definition-term",
            },
            "ref": null,
            "rendered": "species",
            "type": "dt",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "Human",
            },
            "ref": null,
            "rendered": "Human",
            "type": "dd",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "homeworld",
              "className": "card-definition-term",
            },
            "ref": null,
            "rendered": "homeworld",
            "type": "dt",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "Tatooine",
            },
            "ref": null,
            "rendered": "Tatooine",
            "type": "dd",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "language",
              "className": "card-definition-term",
            },
            "ref": null,
            "rendered": "language",
            "type": "dt",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "Galactic Basic",
            },
            "ref": null,
            "rendered": "Galactic Basic",
            "type": "dd",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "population",
              "className": "card-definition-term",
            },
            "ref": null,
            "rendered": "population",
            "type": "dt",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": 200000,
            },
            "ref": null,
            "rendered": 200000,
            "type": "dd",
          },
        ],
        "type": "dl",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="card-header"
>
            <h3
                        className="card-title"
            />
            <Button />
</section>,
          <dl
            className="card-definition-list"
>
            <dt
                        className="card-definition-term"
            >
                        name
            </dt>
            <dd>
                        Luke Skywalker
            </dd>
            <dt
                        className="card-definition-term"
            >
                        species
            </dt>
            <dd>
                        Human
            </dd>
            <dt
                        className="card-definition-term"
            >
                        homeworld
            </dt>
            <dd>
                        Tatooine
            </dd>
            <dt
                        className="card-definition-term"
            >
                        language
            </dt>
            <dd>
                        Galactic Basic
            </dd>
            <dt
                        className="card-definition-term"
            >
                        population
            </dt>
            <dd>
                        200000
            </dd>
</dl>,
        ],
        "className": "card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="card-title"
/>,
              <Button />,
            ],
            "className": "card-header",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": "card-title",
              },
              "ref": null,
              "rendered": null,
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              Array [
                <dt
                  className="card-definition-term"
>
                  name
</dt>,
                <dd>
                  Luke Skywalker
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  species
</dt>,
                <dd>
                  Human
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  homeworld
</dt>,
                <dd>
                  Tatooine
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  language
</dt>,
                <dd>
                  Galactic Basic
</dd>,
              ],
              Array [
                <dt
                  className="card-definition-term"
>
                  population
</dt>,
                <dd>
                  200000
</dd>,
              ],
            ],
            "className": "card-definition-list",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "name",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "name",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Luke Skywalker",
              },
              "ref": null,
              "rendered": "Luke Skywalker",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "species",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "species",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Human",
              },
              "ref": null,
              "rendered": "Human",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "homeworld",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "homeworld",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Tatooine",
              },
              "ref": null,
              "rendered": "Tatooine",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "language",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "language",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "Galactic Basic",
              },
              "ref": null,
              "rendered": "Galactic Basic",
              "type": "dd",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": "population",
                "className": "card-definition-term",
              },
              "ref": null,
              "rendered": "population",
              "type": "dt",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": 200000,
              },
              "ref": null,
              "rendered": 200000,
              "type": "dd",
            },
          ],
          "type": "dl",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
